<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T4general_selector</class>
 <widget class="QDialog" name="T4general_selector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>List of all possible incrementers</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="textLabel_one_more">
     <property name="text">
      <string>Select one or more items</string>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="listBox_filtered"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="15,1,1">
     <item>
      <layout class="QHBoxLayout" name="_2">
       <item>
        <widget class="QLabel" name="textLabel1">
         <property name="text">
          <string>Filter of names</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_filter"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="buttonCancel">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonOk">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="shortcut">
        <string/>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_filter</tabstop>
  <tabstop>listBox_filtered</tabstop>
  <tabstop>buttonCancel</tabstop>
  <tabstop>buttonOk</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonOk</sender>
   <signal>clicked()</signal>
   <receiver>T4general_selector</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>482</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>494</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonCancel</sender>
   <signal>clicked()</signal>
   <receiver>T4general_selector</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>301</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
